MAINTAINERS

```diff
@@ -1,3 +1,15 @@
 Brian Exelbierd <bexelbie@redhat.com> @bexelbie
 Navid Shaikh <nshaikh@redhat.com> @navidshaikh
 Lalatendu Mohanty <lmohanty@redhat.com> @LalatenduMohanty
+
+## How to release the vagrant-service-manager plugin
+
+1. Put the gemfile in pkg/ with `rake build`
```

Comment:
Do we really need "## How to release the vagrant-service-manager plugin" section in  MAINTAINERS  file. IMO it does not belong here.